recipes/Getting_Started/Getting_Started_with_Replicate.ipynb

```diff
@@ -136,10 +136,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "id": "79a5b2e1-f7eb-4de3-bfb9-f049b0c3fe10",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Lookos like we have outputs to clear here

Reply:
also are you handling 401 issue separately, or should we address the error here?

Reply:
Will clear.

Handling the 401 issue separately.